### PR TITLE
feature : Header Website

### PR DESCRIPTION
**Pull request by ardisaurus**
#Revier
@mamunsw 
@novrianta-sbr 

#Description
Create Header homepage on website

#Change
Header component

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>Hello World</h1>
```
Comment (novrianta-sbr) [inline index.html:1]: Ubah huruf Kapital

**Comment by mamunsw** [comment]
Capitalize

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+<p>Ini contoh halaman Kelompok 6</p>
```
Comment (mamunsw) [inline index.html:2]: Capital

**Review by mamunsw — COMMENTED** [review]
Capital

**Comment by ardisaurus** [comment]
Done

**Pull request merged**